src/components/button-more/button-more.js

```diff
@@ -45,12 +45,12 @@ export class ButtonMoreComponent {
     );
   }
 
-  handleItemClickDelete() {
-    this.props.onDeleteClicked();
+  handleItemClickDelete(e) {
+    this.props.onDeleteClicked(e.target);
```

Comment:
it would probably be more logical to pass entire event `e` to `onDeleteClicked `